.editorconfig

```diff
@@ -0,0 +1,18 @@
+# Use editorconfig to stop whitespace wars
```

Comment:
> Use editorconfig to stop whitespace wars

Sounds like loser talk.

Reply:
https://www.youtube.com/watch?v=SsoOG6ZeyUI&feature=youtu.be&t=62s